static/app/components/profiling/ProfilingOnboarding/profilingOnboardingModal.tsx

```diff
@@ -210,8 +212,83 @@ function SelectProjectStep({
   );
 }
 
+function AndroidInstallSteps() {
+  return (
+    <Fragment>
+      <li>
+        <StepTitle>{t('Update your projects SDK version')}</StepTitle>
+        <p>
+          {t('Make sure your SDKs are upgraded to at least version 6.0.0 (sentry-java).')}
```

Comment:
I think should should be the `sentry-android` sdk
```suggestion
          {t('Make sure your SDKs are upgraded to at least version 6.0.0 (sentry-android).')}
```